### PR TITLE
Sean / DP2P QR code

### PR DESCRIPTION
**Pull request by sean-binary**
# Description

Summary

Changes:

-   Added DP2P QR code and copywriting for desktop

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mustofa-binary — APPROVED** [review]

File: src/pages/p2p/components/_availability.js [hunk 1]
```diff
@@ -102,7 +106,24 @@ const Line = styled.div`
...
+    width: 78%;
+`
+
+const Right = styled.div`
+    width: 22%;
+`
+
+const query = graphql`
+    query {
+        qr_code: file(relativePath: { eq: "p2p_playstore.png" }) {
```
Comment (mustofa-binary) [inline src/pages/p2p/components/_availability.js:119]: can we move it to a folder named p2p for example?

File: src/pages/p2p/components/_availability.js [hunk 2]
```diff
@@ -169,24 +196,58 @@ const Availability = () => {
...
-                            />
+                            <Show.Desktop min_width="992">
+                                <Flex>
+                                    <Left>
+                                        <Localize
+                                            translate_text="1. Scan this QR code to download the app from the <0>Google Play Store</0>."
+                                            components={[
+                                                <LocalizedLinkText
+                                                    external
+                                                    to="https://play.google.com/store/apps/details?id=com.deriv.dp2p"
```
Comment (mustofa-binary) [inline src/pages/p2p/components/_availability.js:217]: pls add this to utility strings

File: src/pages/p2p/components/_availability.js [hunk 3]
```diff
@@ -169,24 +196,58 @@ const Availability = () => {
...
+                                    </Right>
+                                </Flex>
+                            </Show.Desktop>
+                            <Show.Mobile>
+                                <Localize
+                                    translate_text="1. Get the app from the <0>Google Play Store</0>."
+                                    components={[
+                                        <LocalizedLinkText
+                                            external
+                                            to="https://play.google.com/store/apps/details?id=com.deriv.dp2p"
```
Comment (mustofa-binary) [inline src/pages/p2p/components/_availability.js:242]: same here

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/p2p/components/_availability.js (hunk 2 above)
Comment (sean-binary) [reply src/pages/p2p/components/_availability.js:217]: Alright

File: src/pages/p2p/components/_availability.js (hunk 3 above)
Comment (sean-binary) [reply src/pages/p2p/components/_availability.js:242]: Alright

File: src/pages/p2p/components/_availability.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/p2p/components/_availability.js:119]: Sure

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**